helm/dex-app/templates/dex-k8s-authenticator/configmap-giantswarm.yaml

```diff
@@ -24,8 +27,8 @@ data:
       k8s_master_uri: https://{{ .Values.Installation.V1.Kubernetes.API.InternalAddress }}
       {{ end }}
       k8s_ca_pem: {{ toYaml .Values.Installation.V1.Secret.Kubernetes.CAPem | indent 8 }}
-      {{- if .Values.Installation.V1.GiantSwarm.OIDC.Enabled }}
```

Comment:
Doesn't removing the dashes cause the presence on unwanted blank lines in the generated file?